### PR TITLE
Test controller changes

### PR DESCRIPTION
**Pull request by arendarenko**
Fix for #101 
Add storeManager class to get default storeId instead of defining it's in code.

### CODE REVIEW

**Review by gonzaloebiz — APPROVED** [review]

**Review by gonzaloebiz — CHANGES_REQUESTED** [review]
Please, see Travis

**Comment by arendarenko** [comment]
Hi @gonzaloebiz 
I fixed all Travis issues, please take a look.

**Comment by arendarenko** [comment]
Hi @gonzaloebiz 
I fixed all travis issues, please take a look.

**Review by gonzaloebiz — APPROVED** [review]

File: Controller/Adminhtml/Email/Test.php [hunk 1]
```diff
@@ -23,21 +24,27 @@ class Test extends \Magento\Backend\App\Action
      */
     protected $_helper;
 
+    /** @var StoreManagerInterface */
+    public $storeManager;
```
Comment (ihor-sviziev) [inline Controller/Adminhtml/Email/Test.php:28]: @alexeya-ven @gonzaloebiz  Shouldn't it be private or at least protected?

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: Controller/Adminhtml/Email/Test.php (hunk 1 above)
Comment (gonzaloebiz) [reply Controller/Adminhtml/Email/Test.php:28]: must be protected

**Pull request merged**